pkg/networkservice/common/excludedprefixes/const.go

```diff
@@ -18,7 +18,10 @@ package excludedprefixes
 
 /* These variables set default path to the config file */
 const (
-	prefixesFile            = "excluded_prefixes.yaml"
-	nsmConfigDir            = "/var/lib/networkservicemesh/config"
-	prefixesFilePathDefault = nsmConfigDir + "/" + prefixesFile
+	// PrefixesFile - excluded prefixes file name
+	PrefixesFile = "excluded_prefixes.yaml"
+	// NsmConfigDir - excluded prefixes file directory name
+	NsmConfigDir = "/var/lib/networkservicemesh/config"
+	// PrefixesFilePathDefault - excluded prefixes file absolute path
+	PrefixesFilePathDefault = NsmConfigDir + "/" + PrefixesFile
```

Comment:
Do we need to export only this constant?

Reply:
And `NsmConfigDir`, because we need to create directory if it doesn't exist.

Reply:
I guess we can don't care about NsmConfigDir because of k8s provides DirectoryOrCreate option for volumes :)